detectable/src/main/java/com/synopsys/integration/detectable/detectables/bitbake/parse/GraphParserTransformer.java

```diff
@@ -54,18 +54,18 @@ public BitbakeGraph transform(final GraphParser graphParser) {
         return bitbakeGraph;
     }
 
-    private String getNameFromNode(final GraphNode graphNode) {
-        final String[] nodeIdPieces = graphNode.getId().split(".do_");
-        return nodeIdPieces[0].replaceAll("\"", "");
+    private String getNameFromNode(GraphNode graphNode) {
+        String[] nodeIdPieces = graphNode.getId().split(".do_");
+        return nodeIdPieces[0].replace("\"", "");
```

Comment:
Actual SonarCloud issue is to use replace to the Pattern.compile cost of replaceAll if we are just escaping java string characters.